spec/amber/router/pipe/pipeline_spec.cr

```diff
@@ -67,6 +67,13 @@ module Amber
           response.body.should eq "Destroy"
         end
 
+        it "perform HEAD request" do
+          request = HTTP::Request.new("HEAD", "/hello/1")
+          response = create_request_and_return_io(pipeline, request)
+          response.headers["Content-Length"].should eq "0"
+          response.body.empty?.should be_true
+        end
```

Comment:
Should we check a positive (200) response when expected and a 404 code as well?

This may end up being scope creep for this PR, as these comments could apply to many of these tests.

Reply:
In that case we should check for all headers, this was more checking if the pipeline was processing the content.

Reply:
I see, I was muddying the waters between pipeline tests and controller tests. I should have looked closer at the naming.